.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
@Soullivaneuh except that your bridge does not have anything with a ``__static`` method needing to be called when the class is loaded. This was the specific need asked in this github issue, not a requirement when writing custom autoloaders.
And your "custom" autoloader will not prevent library conflicts, as it will still delegate calls to the composer autoloader for any class, meaning it will still load everything.